css/style.css

```
@@ -59,6 +73,28 @@ a {
     background-color: var(--grey-200);
     text-transform: uppercase;
     font-weight: 500;
+    outline: 5px solid rgba(0, 81, 255, 0.3);
+    outline-offset: -5px;
+}
+
+.center {
+    width: 1160px;
+    margin: 0 auto;
+    padding: 0 10px;
+}
+
+.main-navigation-wrapper {
+    display: grid;
+    justify-content: stretch;
+    grid-template-columns: min-content 1fr min-content;
+    margin: 50px auto;
```

Можно написать отдельно верх и низ, тогда не придется менять левый и правый. То есть, сделать это так, чтобы не перебивать стили блока `.center`
```suggestion
    margin-top: 50px;
    margin-bottom: 50px;
```